### PR TITLE
Add README for mysql.

### PR DESCRIPTION
**Pull request by revit13**
This PR contains a README for mysql.

### CODE REVIEW

**Review by roytman — APPROVED** [review]

File: fybrik/README_mysql.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
+### Version compatibility matrix
+
+| Fybrik           | AFM     | Command
```
Comment (roytman) [inline fybrik/README_mysql.md:3]: For compatibility reasons all table rows should have borders (pipe symbols) at the start and at the end

The fix

| Fybrik | AFM    | Command                                                                         |
|--------|--------|---------------------------------------------------------------------------------|
| 0.7.x  | 0.1.x  | `https://github.com/fybrik/airbyte-module/releases/download/v0.1.0/module.yaml` |
| 1.0.x  | 0.2.x  | `https://github.com/fybrik/airbyte-module/releases/download/v0.2.0/module.yaml` |
| 1.1.x  | 0.2.x  | `https://github.com/fybrik/airbyte-module/releases/download/v0.2.0/module.yaml` |
| master | master | `https://raw.githubusercontent.com/fybrik/airbyte-module/master/module.yaml`    |

File: fybrik/README_mysql.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+      helm install mysql bitnami/mysql -n fybrik-airbyte-sample
+      ```
+    2. Use the instructions from the helm chart notes to run a pod that is use as a client and connect to the service:
+      ```bash
+      MYSQL_ROOT_PASSWORD=$(kubectl get secret --namespace fybrik-airbyte-sample mysql -o jsonpath="{.data.mysql-root-password}" | base64 -d)
+      kubectl run mysql-client --rm --tty -i --restart='Never' --image  docker.io/bitnami/mysql:8.0.32-debian-11-r0 --namespace fybrik-airbyte-sample --env MYSQL_ROOT_PASSWORD=$MYSQL_ROOT_PASSWORD --command -- bash
+      mysql -h mysql.fybrik-airbyte-sample.svc.cluster.local -uroot -p"$MYSQL_ROOT_PASSWORD"
+      ```
+   
+    3. In mysql client shell insert the following commands to upload fake data into a new database called `fybrik`:
```
Comment (roytman) [inline fybrik/README_mysql.md:58]: In **a** mysql client shell **prompt** ...

File: fybrik/README_mysql.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+      helm install mysql bitnami/mysql -n fybrik-airbyte-sample
+      ```
+    2. Use the instructions from the helm chart notes to run a pod that is use as a client and connect to the service:
+      ```bash
+      MYSQL_ROOT_PASSWORD=$(kubectl get secret --namespace fybrik-airbyte-sample mysql -o jsonpath="{.data.mysql-root-password}" | base64 -d)
+      kubectl run mysql-client --rm --tty -i --restart='Never' --image  docker.io/bitnami/mysql:8.0.32-debian-11-r0 --namespace fybrik-airbyte-sample --env MYSQL_ROOT_PASSWORD=$MYSQL_ROOT_PASSWORD --command -- bash
+      mysql -h mysql.fybrik-airbyte-sample.svc.cluster.local -uroot -p"$MYSQL_ROOT_PASSWORD"
+      ```
+   
+    3. In mysql client shell insert the following commands to upload fake data into a new database called `test`:
```
Comment (roytman) [inline fybrik/README_mysql.md:126]: as above
In a mysql client shell prompt

File: fybrik/README_mysql.md (hunk 1 above)
Comment (revit13) [reply fybrik/README_mysql.md]: OK thanks. I updated the matrix in `README.md` and removed it from `README_mysql.md`

File: fybrik/README_mysql.md (hunk 2 above)
Comment (revit13) [reply fybrik/README_mysql.md]: Fixed Thanks

File: fybrik/README_mysql.md (hunk 3 above)
Comment (revit13) [reply fybrik/README_mysql.md]: Fixed Thanks

**Pull request merged**